package.json

```diff
@@ -20,16 +20,20 @@
     "@react-pdf/renderer": "3.1.12",
     "@sentry/react": "7.55.2",
     "@sentry/tracing": "7.55.2",
+    "@types/file-saver": "2.0.5",
```

Comment:
Types are a development dependency. Move this to the `devDependencies` section. 

Remember when adding a dev dependency you can use the `-D` option with `yarn add` like this: `yarn add -D @types/file-saver`